extensions/podman/packages/extension/src/podman-info-helper.spec.ts

```diff
@@ -125,8 +124,15 @@ test('should grab podman from the installer', async () => {
   expect(records.podmanMachineNetworkBackend).toBe('netavark');
   expect(records.podmanMachineNetworkBackendVersion).toBe('netavark 1.7.0');
   expect(records.podmanMachineNetworkOciRuntime).toBeDefined();
-  expect((records.podmanMachineNetworkOciRuntime as any).name).toBe('crun');
```

Comment:
wondering if podmanMachineNetworkOciRuntime should not be with a type containing {name?: unknown} or {name?: string} in the podman-info-helper class

Reply:
or have podmanMachineNetworkOciRuntimeName property